example/mage.go

```diff
@@ -10,8 +10,8 @@ import (
 func init() {
 	targets.ProjectName = "example"
 	targets.ProjectType = "service"
-	targets.DockerComposeDevDependencies = []string{"redis", "cockroach", "minio", "rabbit"}
-	targets.DockerComposeTestDependencies = []string{"cockroach"}
```

Comment:
I feel as an example its more useful to clearly show these dependencies can be different?

Reply:
☝️

Reply:
Added in 👍